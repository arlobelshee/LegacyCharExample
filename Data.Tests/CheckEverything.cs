using ApprovalTests;
using ApprovalTests.Reporters;
using Data.PipelineSynchronous;
using NUnit.Framework;

namespace Data.Tests
{
	[TestFixture]
	[UseReporter(typeof(VisualStudioReporter))]

	public class CheckEverything
	{
		[Test]
		public void TestPipeline()
		{
			var pipeSource = DoEverything.MakePipe(out var characterCollector, out var configCollector);
			Approvals.Verify( pipeSource.ToNewString());
		}

	}
}
