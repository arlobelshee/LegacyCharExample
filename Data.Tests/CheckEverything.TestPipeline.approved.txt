--> CharacterFile CharacterFile.From(String fileName)
 |--> ConfigFile ConfigFile.Matching(CharacterFile characterFile)
 | |--> Collector
 |--> Collector
